platform/polkadot/client.go

```diff
@@ -12,7 +12,7 @@ type Client struct {
 
 func (c *Client) GetTransfersOfAddress(address string) ([]Transfer, error) {
 	var res SubscanResponse
-	err := c.Post(&res, "/scan/transfers", TransfersRequest{Address: address, Row: blockatlas.TxPerPage})
+	err := c.Post(&res, "scan/transfers", TransfersRequest{Address: address, Row: blockatlas.TxPerPage})
```

Comment:
I think we should remove / in `GetBase`? what if there is a trailing / in `config.yml`

Reply:
the method applies the `/`, we don't need them